terrautils/spatial.py

```diff
@@ -0,0 +1,244 @@
+"""Spatial
+
+This module provides useful methods for spatial referencing.
+"""
+
+
+import utm
+
+# FORMAT CONVERSION -------------------------------------
+def calculate_bounding_box(gps_bounds, z_value=0):
+    """Given a set of GPS boundaries, return array of 4 vertices representing the polygon.
+
+    gps_bounds -- (lat(y) min, lat(y) max, long(x) min, long(x) max)
+
+    Returns:
+        List of 4 polygon vertices [ul, ur, lr, ll] without z values.
+    """
+
+    return [
+        (gps_bounds[2], gps_bounds[1], z_value), # upper-left
+        (gps_bounds[3], gps_bounds[1], z_value), # upper-right
+        (gps_bounds[3], gps_bounds[0], z_value), # lower-right
+        (gps_bounds[2], gps_bounds[0], z_value)  # lower-left
+    ]
+
+
+def calculate_centroid(gps_bounds):
```

Comment:
Just want to make sure that we aren't re-inventing the wheel and that we are taking advantage of existing geospatial libraries where they exist  e.g. https://shapely.readthedocs.io/en/latest/manual.html#object.centroid https://shapely.readthedocs.io/en/latest/manual.html#object.union

Not necessary for beta release, but something to keep in mind b/c the canonical implementation is more reliable and easier to maintain. 

@jterstriep could you please comment?